### ISSUE TITLE
Add "Flags" as icons in the facility manager, maps and sheets and new views 

### ISSUE THREAD

**Issue by bluenevus** [MicroHealthLLC/mGis#935]
The flags will be present if its overdue, on watch, has related task, has related issue, complete, has attachment  and (risks when that comes in).  Each shall have its own icon representative.  the overdue will reuse the current overdue icon.  Hovering over any of these icons shall display the titles of what constitutes the flag.  Clicking on the specific title shall take them to that item.  For sheets, we will remove the overdue and on watch columns.  Add a column after task called Flags and show these icons.  The exports will have the same icons in pdf and equivalent (.e.g abbreviations)  in excel understanding the limitations of excel exports

**Comment by juan-rivera-2020** [comment]
Working this now. Functionality is complete.  Do we still want the **Flags** column as explained in the original description? Demo to follow for evaluation.

**Comment by bluenevus** [comment]
yes so we can get space back on the sheets view also on the manager and map view

**Comment by juan-rivera-2020** [comment]
Sounds good.

**Comment by juan-rivera-2020** [comment]
I added the box to the On Watch column but scrapped it to better plan the feature as you described it originally.  
![image](https://user-images.githubusercontent.com/72885316/99412448-b9befc80-28c2-11eb-82ef-a118301a7bf7.png)

**Comment by bluenevus** [comment]
that looks good

**Comment by bluenevus** [comment]
if its going to be a while, can I get what you did in that image above in sheets?

**Comment by JuanCodeAtATime** [comment]
Okay.  Yes...When I initially took that image, the functionality did  it work consistently throughout the pages, so I moved onto another enhancement....but I’ll get back on that task and finish it.
